.circleci/config.yml

```diff
@@ -57,5 +86,9 @@ workflows:
   version: 2
   build_and_test:
     jobs:
-      - build
+      - lint
       - test
```

Comment:
can you make the `test` stage require `lint`?

Reply:
Since both are jobs that run under 5 min, this would reveal quicker any problem and would show both linting and testing. Having them sequentially would mean that we would go through first the linting problems and then the testing ones.

Is there a reason or improvement to run them sequentially?

Reply:
I'm inclined to agree with @cpanato -- if the linting steps fail, there's probably no reason to think the testing step will be valid anyway: there's a decent chance it will fail for the same reasons.

Reply:
In that case, should I run the linting and testing in the same job, there wouldn't be any gain in running them separately, would it?

Reply:
I put both steps as one, if we are not going to speed up showing as much errors, there is no point in penalising by having them in separate containers.